kinematics_base/include/moveit/kinematics_base/kinematics_base.h

```diff
@@ -56,6 +56,48 @@ class RobotState;
 namespace kinematics
 {
 
+/*
+ * @enum DiscretizationMethods
+ *
+ * @brief Flags for choosing the type discretization method applied on the redundant joints during an ik query
+ */
+namespace DiscretizationMethods
```

Comment:
in C++ non-class enums expose the values outside the scope of the enum type, so in order to get something like Foo::Value, using namespaces is the nice thing to do.